frontend/web/components/TestWebhook.js

```diff
@@ -43,6 +43,7 @@ export default class TheComponent extends Component {
               {error && <ErrorMessage error={this.state.error}/>}
               {success && <SuccessMessage message="Your API returned with a successful 200 response."/>}
               <Button
+                type="button"
```

Comment:
Since the type wasn't defined, clicking this button was acting like a form submit and hence saving / closing the modal.